### ISSUE TITLE
build error with MRB_ENABLE_ALL_SYMBOLS

### ISSUE THREAD

**Issue by takahashim** [mruby/mruby#5187]
I got an error with `conf.cc.defines = %w(MRB_ENABLE_ALL_SYMBOLS)` in build_config.

```
$ rake
CC    mrbgems/mruby-bin-mrbc/tools/mrbc/mrbc.c -> build/host/mrbgems/mruby-bin-mrbc/tools/mrbc/mrbc.o
CC    src/object.c -> build/host/src/object.o
CC    src/kernel.c -> build/host/src/kernel.o
CC    src/print.c -> build/host/src/print.o
CC    src/class.c -> build/host/src/class.o
CC    src/array.c -> build/host/src/array.o
CC    src/etc.c -> build/host/src/etc.o
CC    src/state.c -> build/host/src/state.o
CC    src/init.c -> build/host/src/init.o
CC    src/vm.c -> build/host/src/vm.o
CC    src/string.c -> build/host/src/string.o
CC    src/hash.c -> build/host/src/hash.o
CC    src/codedump.c -> build/host/src/codedump.o
CC    src/debug.c -> build/host/src/debug.o
CC    src/range.c -> build/host/src/range.o
CC    src/fmt_fp.c -> build/host/src/fmt_fp.o
CC    src/dump.c -> build/host/src/dump.o
CC    src/error.c -> build/host/src/error.o
CC    src/backtrace.c -> build/host/src/backtrace.o
CC    src/numeric.c -> build/host/src/numeric.o
CC    src/enum.c -> build/host/src/enum.o
CC    src/symbol.c -> build/host/src/symbol.o
/tmp/xxx/mruby/src/symbol.c:327:38: error: no member named 'symbuf' in 'struct mrb_state'
  return sym2name_len(mrb, sym, mrb->symbuf, lenp);
                                ~~~  ^
1 error generated.
rake aborted!
```

**Comment by matz** [comment]
It's not too hard to fix this issue but considering #5116 we might need to remove `MRB_ENABLE_ALL_AYMBOLS` altogether.
Hmm.

**Comment by shuujii** [comment]
@takahashim Thank you for your report.

I've noticed that this issue is due to #5166 (originally it was working by accident because it was typo), but at the moment mruby doesn't work if `MRB_ENABLE_ALL_SYMBOLS` is specified anyway (#5116), so I neglect this issue.

If you want to fix this issue for the time being, I think you can apply [this fix](https://github.com/shuujii/mruby/commit/a7b50be35be6d594423290043f68822f54593c5a).

**Comment by matz** [comment]
@shuujii Could you make the fix a pull-request?

**Comment by shuujii** [comment]
> Could you make the fix a pull-request?

Sure.

**Comment by matz** [comment]
closed by #5189

**Comment by takahashim** [comment]
I confirmed to build successfully with master. Thanks for fix.
